test/filemagic_test.rb

```diff
@@ -133,7 +139,6 @@ def test_compile
     res = fm.compile(path_to('perl'))
     fm.close
     assert(res)
-    File.unlink(path_to('perl.mgc', '.'))
```

Comment:
Ah, sorry but maybe I accidentally removed that file and misunderstood root cause... >_<